lepton/qemu.go

```diff
@@ -66,32 +141,30 @@ func (q *qemu) Start(rconfig *RunConfig) error {
 }
 
 func (q *qemu) Args(rconfig *RunConfig) []string {
-	// TODO : this should come from q.drives and ifaces
+	// todo create a qemu from serial
 	args := []string{}
-
-	boot := []string{"-drive", "file=image,format=raw,index=0"}
-	storage := []string{"-drive", "file=image,format=raw,if=virtio"}
-	var net []string
-	if rconfig.Bridged {
-		net = []string{"-device", "virtio-net,mac=7e:b8:7e:87:4a:ea,netdev=n0",
```

Comment:
@tijoytom did you mean for this to have (down)script=no?

Reply:
OK, I see what happened here. For the 'user' device type the script parameters are invalid. I added that logic to the `netdev` String() method and added a unit test. PR should be good now. populating a `qemu` struct with info from a `RunConfig` should be enough to render a good command string.  `-nodefaults`, `-no-reboot`, `-device isa-debug-exit`, I'm guessing these things are not variable and will be used in all cases, so they are left hard-coded.

Reply:
you can leave them hard-coded.